lib/two_factor_authentication.rb

```diff
@@ -1,9 +1,9 @@
 require 'two_factor_authentication/version'
 require 'devise'
 require 'active_support/concern'
-require "active_model"
-require "active_support/core_ext/class/attribute_accessors"
-require "cgi"
+require 'active_model'
```

Comment:
PR #108 added an Activerecord dependency here that I removed in the merge. Unknown if it's needed but it doesn't seem to be and tests pass.